tests/contrib/minikube/test_kubernetes_pod_operator.py

```diff
@@ -251,6 +251,22 @@ def test_faulty_service_account(self):
         with self.assertRaises(ApiException):
             k.execute(None)
 
+    def test_faulty_image_pull_secrets(self):
+        bad_image_pull_secrets = "foobar"
+        k = KubernetesPodOperator(
+            namespace='default',
+            image="ubuntu:16.04",
+            cmds=["bash", "-cx"],
+            arguments=["echo 10"],
+            labels={"foo": "bar"},
+            name="test",
+            task_id="task",
+            startup_timeout_seconds=5,
+            image_pull_secrets=bad_image_pull_secrets
+        )
+        with self.assertRaises(ApiException):
```

Comment:
This test alone seems a little bit hard-to-reason about.

Could you do something like `assertRaisesRegexp(ApiException, r'invalid pull secrets')` (or what ever string indicates the pull secrets are the problem - try to make it not too specific.) And perhaps a comment too so we know what this test was trying to test if it fails in the future.

Reply:
Thanks, anyway this doesn't work apparently, if it is not needed, the pull secret is not used it seems, I will think of another way to test this